src/components/postgresConnect.ts

```diff
@@ -123,6 +125,46 @@ class PostgresDB {
     }
   }
 
+  public getActionsProcessing = async (templateId: number): Promise<ActionQueue[]> => {
+    const text =
+      "SELECT id, action_code, trigger_payload, parameter_queries FROM action_queue WHERE template_id = $1 AND status = 'Processing' ORDER BY sequence"
+    try {
+      const result = await this.query({
+        text,
+        values: [templateId],
+      })
+      return result.rows as ActionQueue[]
+    } catch (err) {
+      throw err
```

Comment:
Was just looking at this file, and was wondering about catch(err) and throw err, is it there for future sakes ? when we want to handle errors in a more specific way ?

Reply:
Yes, we can add more specific error messaging at some point, but my understanding is that this would simply pass any error thrown by the `node-postgres` module back up to the calling method, which can decide how to handle it. Is that correct?

Reply:
Yeah, I guess what I was trying to say if the error is caught and then thrown straight away, it's the same as not having the try/catch.

Reply:
Okay, point taken. Since this is done like this in a lot of places, let's re-factor them all later on once we have a better strategy for error handling?